README.md

```diff
@@ -1,15 +1,245 @@
+--ref /path/to/reference.fasta
+```
+
+
+### 3. Clone this repository 
+
+Download the code contained in this repository with: 
+
+```
+git clone https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf
+```
+
+This will create a directory with the following structure: 
+```
+Somatic-shortV-nf/
+├── LICENSE
+├── README.md
+├── config/
+├── images/
+├── main.nf
+├── modules/
+├── PBS_gadi_runPipeline.sh
+└── nextflow.config
+```
+The important features are: 
+
+* **main.nf** contains the main nextflow script that calls all the processes in the workflow.
+* **nextflow.config** contains default parameters to use in the pipeline.
+* **modules** contains individual process files for each step in the workflow. 
+* **config** contains infrastructure-specific config files (this is currently under development)
+
+### 4. Run the pipeline 
+
+The most basic run command for this pipeline is: 
+
+```
+nextflow run main.nf --ref reference.fasta
+
+```
+
+By default, this will generate `work` directory, `results` output directory and a `runInfo` run metrics directory in the same location you ran the pipeline from. 
+
+To specify additional optional tool-specific parameters, see what flags are supported by running:
+
+```
+nextflow run main.nf --help 
+```
+
+If for any reason your workflow fails, you are able to resume the workflow from the last successful process with `-resume`. 
+
+
+### 5. Results 
+Once the pipeline is complete, you will find all outputs for each sample in the results directory. **Note** I still need to delete arrange the output files in proper sub-directory structure.
+
+## Infrastructure usage and recommendations
+
+This pipeline has been successfully implemented on NCI Gadi  using infrastructure-specific configs. These configs can be used to interact with the job scheduler and assign a project code to all task job submissions for billing purposes. You can use the following flags to handle accounting:
+
+* `--whoami` your NCI or Pawsey user name
+* `--gadi-account` the Gadi project account you would like to bill service units to
+
+
+### NCI Gadi HPC
+
+Before running the pipeline you will need to load Nextflow and Singularity, both of which are globally installed modules on Gadi. You can do this by running the commands below:
+
+```
+module purge
+module load nextflow singularity
+```
+
+To execute this workflow on NCI Gadi HPC, you will need to specify the following flags to the default run command:
+
+```
+nextflow run main.nf --input sample.tsv --ref /path/to/ref --gadi-account <account> --whoami <username> -profile gadi
+```
+
+Please be aware that as of October 2023, NCI Gadi HPC queues do not have external network access. This means you will not be able to pull the workflow code base or containers if you submit your nextflow run command as a job on any of the standard job queues. NCI currently recommends you run your Nextflow head job either in a GNU screen or tmux session from the login node or submit it as a job to the copyq.
```

Comment:
Update the date.

Reply:
Should we update the entire para:

Run the main.nf on copyq (this will download all the containers - first time) and then assign normal queues to individual resources?

Reply:
You have a gadi script, you can just tell users to run that.